pkg/handler/processor/dsse/dsse_test.go

```diff
@@ -97,8 +97,8 @@ var (
 	}
 	ite6SLSADoc = processor.Document{
 		Blob:   []byte(ite6SLSA),
-		Type:   processor.DocumentITE6Generic,
```

Comment:
Can we remove `DocumentITE6Generic` type also as we do not want anyone else to use it.

Reply:
i think we can but i think its still useful as a catch all where then we start to see errors come up to some ITE6 things we dont handle as part of https://github.com/guacsec/guac/blob/main/pkg/handler/processor/guesser/type_ite6.go#L31

Reply:
Should we error at that level? Because it will go to the parser and error there instead. We should catch it here.

Reply:
i think the question is whether we want the error message to say "Document with unknown type" or "ITE6 document failed to parse". I feel like the later provides more contextual information.

Reply:
yeah, makes sense.